libs/pilight/protocols/GPIO/ds18b20.c

```diff
@@ -133,6 +139,27 @@ static void *ds18b20Parse(void *param) {
 								}
 								memset(content, '\0', bytes+1);
 
+								if (uname(&unameData) != -1) {
+									char **varr = NULL;
+									int toks = explode(unameData.release, ".", &varr);
```

Comment:
Have you checked for memory leaks with valgrind? You aren't freeing the `&varr` anywhere?